cset-workflow/flow.cylc

```diff
@@ -4,42 +4,53 @@ title = CSET
 description = Workflow for running CSET.
 URL = https://metoffice.github.io/CSET
 
-[scheduler]
-    UTC mode = True
+{% from "jinja_utils" import get_models, glob, max, min, zip, restructure_field_list, sanitise_task_name %}
+{% set models = get_models(ROSE_SUITE_VARIABLES) %}
 
 [scheduling]
     runahead limit = P{{CSET_RUNAHEAD_LIMIT}}
-    initial cycle point = {{CSET_INITIAL_CYCLE_POINT}}
-    final cycle point = {{CSET_FINAL_CYCLE_POINT}}
+
+    # Initial and final cycle points cover the entire period of interest.
+    {% if CSET_CYCLING_MODE == "case_study" %}
+    initial cycle point = {{ min(CSET_CASE_DATES) }}
```

Comment:
DO I understand correctly that this orders the cycle datas in an anscednign order? The final cycle date is the final case study? This means that we do not set the final cycle point per run I assume, but ick each case study based on their file name and then run all the data in that file?

Reply:
Yes, the cases will be processed in ascending order, though due to parallelism they may complete in a slightly different order. No final or initial cycle point needs to be set, as they are just taken from the first and last case dates. For trials you effectively directly set the initial and final cycle points.

The dates are just provided as a list in rose edit, and the file path is used to select the data relevant for that case.